clidocstool_md.go

```diff
@@ -153,6 +153,9 @@ func mdCmdOutput(cmd *cobra.Command, old string) (string, error) {
 		fmt.Fprint(b, "\n\n")
 	}
 
+	// necessary to merge persistent flags
+	cmd.LocalFlags()
+
 	hasFlags := cmd.Flags().HasAvailableFlags()
 
 	cmd.Flags().AddFlagSet(cmd.InheritedFlags())
```

Comment:
Should we  (instead of calling `cmd.LocalFlags()` and  discarding the results, replace these with `cmd.LocalFlags()` ?

```suggestion
	hasFlags := cmd.LocalFlags().HasAvailableFlags()

	cmd.LocalFlags().AddFlagSet(cmd.InheritedFlags())
```

(Or perhaps store it in a variable?)

Reply:
Yes actually better to use it only for flags detection